Tests/GoodCombineExtensionsTests/GoodCombineExtensionsTests.swift

```diff
@@ -0,0 +1,41 @@
+import XCTest
+import CombineExt
+import GoodCombineExtensions
+import GRCompatible
+import Combine
+
+#if !os(macOS)
+
+import UIKit
+
+final class GoodCombineExtensionsTests: XCTestCase {
+
+    func testButtonPublisher() {
+        var cancellables = Set<AnyCancellable>()
+        let exp = expectation(description: "Waiting for buttonclick")
+
+        let button = UIButton()
+        var values = [Int]()
+        button.gr.publisher(for: .touchUpInside)
+            .handleEvents(receiveSubscription: { _ in
+                values.append(1)
+                exp.fulfill()
+            })
+            .sink { _ in
+                print("Nope")
+            }
+            .store(in: &cancellables)
+
+        button.sendActions(for: .touchUpInside)
+
+        waitForExpectations(timeout: 5)
+        XCTAssertEqual(values, [1], "Values should contain a one")
```

Comment:
Asi by som dal logickejsi text :). Button publisher not subscribe/Button publisher extension not working. etc. Nech vieme dovod preco ten test padol ak padol :) To ze to nieje 1 nieje uplne test scenar a to co chceme dostat na vystupe :)